ethToElrond/v2/bridge/bridgeExecutor.go

```diff
@@ -259,8 +259,8 @@ func (executor *bridgeExecutor) SignActionOnElrond(ctx context.Context) error {
 	return nil
 }
 
-// IsQuorumReachedOnElrond returns true if the proposed transfer reached the set quorum
-func (executor *bridgeExecutor) IsQuorumReachedOnElrond(ctx context.Context) (bool, error) {
+// ProcessQuorumReachedOnElrond returns true if the proposed transfer reached the set quorum
+func (executor *bridgeExecutor) ProcessQuorumReachedOnElrond(ctx context.Context) (bool, error) {
```

Comment:
the ProcessQuorumReachedOnElrond/Ethereum were given when the action altered the inner state of the bridge. Now, it seems completely legit to have it IsQuorumReachedOn.... 
Can be done in another PR